telebot/__init__.py

```diff
@@ -4558,6 +4558,8 @@ def set_sticker_set_thumb(
         :rtype: :obj:`bool`
         """
         return apihelper.set_sticker_set_thumb(self.token, name, user_id, thumb)
+    
+    set_sticker_set_thumb = set_sticker_set_thumbnail
```

Comment:
I propose make it a real method that call set_sticker_set_thumbnail with deprecetion warning.